### DISCUSSION TITLE
Checkmate2 result changes when running multiple times

### DISCUSSION THREAD

**Discussion by physicsbamboo** [CheckMATE2/checkmate2#31] [General]
Hey I noticed that for some sets of parameters, checkmate2 is giving me different results(sometimes "Excluded" but sometimes "Accepted") when executing for multiple times with the exactly the same parameters. Anyone also met this problem?

Thanks!

**Comment by krolbiecki** [comment]
Hi,
sorry I missed this question. in general this is because many things happen in a random way (eg jet and lepton tagging). So if the parameter point is close to the limit between accepted and excluded it will have different result  for different runs. To make sure the results are reproducible, you can use fixed randomseed with -rs some_number_int.
